lib/platform-stats.js

```diff
@@ -11,15 +13,16 @@ const debug = createDebug('spark:platform-stats')
 export const updatePlatformStats = async (pgClient, honestMeasurements, allMeasurements) => {
   const participants = new Set(honestMeasurements.map(m => m.participantAddress))
   await updateDailyParticipants(pgClient, participants)
-  await updateDailyStationStats(pgClient, honestMeasurements, allMeasurements)
+  await updateDailyStationStats(pgClient, new Date(), honestMeasurements, allMeasurements)
 }
 
 /**
  * @param {import('pg').Client} pgClient
+ * @param {Date} day
  * @param {import('./preprocess.js').Measurement[]} honestMeasurements
  * @param {import('./preprocess.js').Measurement[]} allMeasurements
  */
-export const updateDailyStationStats = async (pgClient, honestMeasurements, allMeasurements) => {
+export const updateDailyStationStats = async (pgClient, day, honestMeasurements, allMeasurements) => {
```

Comment:
I have mixed feelings about introducing the `today` parameter to `updateDailyStationStats()`, but I also don't have any good counter-arguments, so let's give this a try and see how it works in practice.

Reply:
Since it's always today in production, and only a different day in tests, can we add this as an optional parameter instead?

Reply:
I also didn't love it at first, but the alternative is fully mimicking the entire updateDailyStationStats function with control over the date to be used in tests (because it has somewhat nuanced logic compared to other insertion-handling functions we have)

Used an optional parameter in https://github.com/filecoin-station/spark-evaluate/pull/278/commits/3b8b7416a86cb7cfe21fd52ca6f45c8c8b8c6f08